HIRS_AttestationCA/src/main/java/hirs/attestationca/service/SupplyChainValidationServiceImpl.java

```diff
@@ -288,6 +303,41 @@ private SupplyChainValidation validatePlatformCredentialAttributes(final Platfor
         }
     }
 
+    private SupplyChainValidation validateDeltaPlatformCredentialAttributes(
+            final PlatformCredential delta,
+            final DeviceInfoReport deviceInfoReport,
+            final PlatformCredential base) {
+    /*
+     * Do we need a new ValidationType for deltas?
```

Comment:
Think a new validation type would be helpful.

Reply:
Added a new type for delta attributes.  I think the existing ValidationType for (base) platform credentials can extend to deltas as well so I did not create a new enum element for delta platform credentials @cyrus-dev